test/model_validation_test.py

```diff
@@ -115,6 +118,62 @@ def test_validation_fails_on_missing_key(self):
             with self.subTest(key=key):
                 test_dict = self.raw_dict.copy()
                 test_dict.pop(key)
-                element = examinee.BasicCredentials(test_dict)
+                element = concourse.BasicCredentials(test_dict)
                 with self.assertRaises(ModelValidationError):
                     element.validate()
+
+
+class AwsModelTest(unittest.TestCase):
```

Comment:
I think it might be less boiler-platy to just use functions to write those tests (as opposed to the old-style junit-like class-based approach)

just use e.g.

```
minimal_dict = {.......}

def test_validation_fails_on_missing_required_key():
  ....
```